### DISCUSSION TITLE
Keyboard key + does not work when rewriting segment time and i can not write right after creating new segment

### DISCUSSION THREAD

**Discussion by slrslr** [mifi/lossless-cut#1759] [Ideas]
Hi,

I have an idea/feedback.

When I am rewriting the digits of the segment/cut start/end time, then the keyboard key "+" does not work to create new segment. It works when i click outside of the time text area (for example into list of segments). If there is any working shortcut, then it is not mentioned on Help/Keyb. shortcuts window nor it is mentioned in the mouse hover tooltip when i mouse aim on the + (new segment button).

After new segment is created, i would want to immediately start re-writing the time of the segment start, but that text area is not active apparently and i have to aim mouse and click.

If you think these 2 things may be improved, please do. Thank you

**Comment by mifi** [comment]
thanks. I've added your UX suggestion here #189
